lib/TextBox.js

```diff
@@ -452,3 +454,13 @@ TextBox.prototype.setSelection = function(container, offset) {
   selection.removeAllRanges();
   selection.addRange(range);
 };
+
+// helpers //////////
+
+function normalizeEndOfLineSequences(string, endOfLineSequence) {
+  if (isUndefined(endOfLineSequence)) {
```

Comment:
How can `endOfLineSeuqence` ever be undefined here?

Reply:
What this does is effectively default to LF. I can remove that switch if it's considered unnecessary.

Reply:
How does that second parameter sometimes exist? Is it `'FOO'.replace(/OO/, function(o, o) { })` ?

Reply:
The way we use it (`text = normalizeEndOfLineSequences(text);`) it never exists, therefore it defaults to LF.

Reply:
That one is super confusing to be honest. Let's always default to `\n` if that does not break anything. The second parameter is completely unused, too. I'd suggest to get rid of it.

Reply:
Ok, will do! 👍🏻

Reply:
Done. 👍🏻